exchanges/deribit/deribit_types.go

```diff
@@ -165,8 +165,13 @@ type FundingRateHistory struct {
 
 // HistoricalVolatilityData stores volatility data for requested symbols
 type HistoricalVolatilityData struct {
-	Timestamp float64
-	Value     float64
+	Timestamp types.Time
+	Value     types.Number
+}
+
+// UnmarshalJSON  parses volatility data from a JSON array into HistoricalVolatilityData fields.
+func (h *HistoricalVolatilityData) UnmarshalJSON(data []byte) error {
+	return json.Unmarshal(data, &[2]any{&h.Timestamp, &h.Value})
```

Comment:
Out of scope: Can you do a quick benchmark for a diff between allocating an array [2] vs just allocating a slice. If you don't get to it, that is fine.

Reply:
<img width="844" alt="Screenshot 2025-05-29 at 11 20 24 at night" src="https://github.com/user-attachments/assets/747d8078-9eea-43ae-b8e7-e47b8b416048" />